templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% block title %}
+  <title>Taxi service</title>
+  {% endblock %}
+  <meta charset="urf-8">

```

Comment:
There is a typo in the charset attribute. It should be `utf-8` instead of `urf-8`.